zeppelin-web/src/index.html

```diff
@@ -173,6 +173,8 @@
     <script src="bower_components/MathJax/MathJax.js"></script>
     <script src="bower_components/clipboard/dist/clipboard.js"></script>
     <script src="bower_components/ngclipboard/dist/ngclipboard.js"></script>
+    <script src="bower_components/scrollMonitor/scrollMonitor.js"></script>
```

Comment:
Is it possible to use `scrollMonitor` without installing it? it's not added to bower.js.